### PR TITLE
Add feedback link to debugger

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
Reimplements #3097

### CODE REVIEW

File: addons/debugger/userscript.js [hunk 1]
```diff
@@ -342,14 +343,44 @@ export default async function ({ addon, global, console, msg }) {
...
+    }
+  }
+  class FeedbackWrapper {
+    constructor() {
+      const feedbackLog = new LogWrapper();
+      feedbackLog.append(
+        new LogText(
+          `${msg("feedback-log")} <a href="https://scratchaddons.com/feedback" target="_blank">${msg(
+            "feedback-log-link"
+          )}</a>`,
```
Comment (TheColaber) [inline addons/debugger/userscript.js:372]: safeMsg pls

Comment (TheColaber) [reply addons/debugger/userscript.js:372]: also, im not sure this works in every language... to separate a sentence

File: addons-l10n/en/debugger.json [hunk 2]
```diff
@@ -14,5 +14,7 @@
   "debugger/block-breakpoint": "breakpoint",
   "debugger/block-log": "log %s",
   "debugger/block-warn": "warn %s",
-  "debugger/block-error": "error %s"
+  "debugger/block-error": "error %s",
+  "debugger/feedback-log": "We'd appreciate it if you'd share how you use this addon",
```
Comment (FunctionalMetatable) [inline addons-l10n/en/debugger.json:18]: Maybe a prefix of `info: `?

File: addons/debugger/userscript.js (hunk 1 above)
Comment (FunctionalMetatable) [reply addons/debugger/userscript.js:372]:
```js
// maybe:
msg("feedback-log", { logLink: `<a href="https://scratchaddons.com/feedback" target="_blank">${msg("feedback-log-link")}</a>` })
```
```js
// for the locales
{
  ...
  "debugger/feedback-log": "We'd appreciate it if you'd share how you use this addon {logLink}",
  "debugger/feedback-log-link": "here"
}

File: addons-l10n/en/debugger.json (hunk 2 above)
Comment (GrahamSH-LLK) [reply addons-l10n/en/debugger.json:18]: That sounds weird

**Comment by WorldLanguages** [comment]
Lol I got stuck on how to properly use safeMsg and/or msg to have the anchor tag be a direct child of `div.log > span`

File: addons/debugger/userscript.js [hunk 3]
```diff
@@ -342,20 +342,52 @@ export default async function ({ addon, global, console, msg }) {
...
+    const s = document.createElement("span");
+    s.innerHTML = safeMsg("feedback-log", { logLink: "$0" }).replace(
+      "$0",
+      Object.assign(document.createElement("a"), {
+        href: "https://scratchaddons.com/feedback?version=1.18-debugger",
+        className: "sa-debugger-feedback",
+        target: "_blank",
+        textContent: msg("feedback-log-link"),
+      }).outerHTML
+    );
```
Comment (WorldLanguages) [inline addons/debugger/userscript.js:369]: Is this the proper way to do it? @apple502j

**Pull request merged**